src/fontMetrics.js

```diff
@@ -198,10 +198,15 @@ const getCharacterMetrics = function(
     let ch = character.charCodeAt(0);
     if (character[0] in extraCharacterMap) {
         ch = extraCharacterMap[character[0]].charCodeAt(0);
-    } else if (cjkRegex.test(character[0])) {
+    } else if (supportedCodepoint(ch) && !metricMap[font][ch]) {
```

Comment:
Note the `&& !metricMap` bit I added on this line... I was thinking that this allows the addition of other font metrics in the future. Not sure whether that makes sense, though.

Reply:
Actually, I think I need to think some more about this.  In Devangari (and probably most of these other Brahmic scripts) there are a lot of combining characters which generally turn into ligatures and have no intrinsic width on their own.  Using a full em width is wrong given that somewhere around half of the characters are zero width vowels.

At a minimum I should choose a narrower character for Brahmic scripts. Really though, I think we can't really measure characters from those scripts in isolation: to actually be accurate we have to measure the whole string of them. I haven't looked yet at how this code is used, but I worry that this is a larger architectural change required here for complete accuracy.

For our needs at KA we only need to be able to display these characters in plain \text{} environments. Not trying to center them above a big capital sigma character or anything, so I'd guess that super-accurate measurements are not all that necessary.

Reply:
The only place we actually use the `width` of any glyph is when we're positioning accent glyphs.  We still need some sort of `height`/`depth` for these so what you've done here I think is okay for now.